### DISCUSSION TITLE
Support label text below icons in Stepper component

### DISCUSSION THREAD

**Discussion by mcmxcdev** [mantinedev/mantine#2139] [Feature requests]
I have a business requirement for a `Stepper` with text below the icons.
Additionally, the text should be a different color for steps that are not yet completed.

I found a good example here: 

![Screenshot from 2022-08-16 11-18-47](https://user-images.githubusercontent.com/16797721/184844688-0a46d773-9969-472a-8ec4-e84c71d509f6.png)

**Comment by bzitzow-osh** [comment]
+1

**Comment by yjyjyjy** [comment]
Will be nice to have!

**Comment by angelinashepherd** [comment]
+10000!!

**Comment by angelinashepherd** [comment]
would also be great to not change the icon upon completion

**Comment by jdpnielsen** [comment]
@rtivital Would you be open to contributions on this?

**Reply by rtivital** [reply]
It is in the backlog, once I get to it, I'll add it in a minor release myself

**Comment by rtivital** [comment]
Implemented this feature, will be available in 9.6 release.
